### PR TITLE
very small and minor fixes

### PR DESCRIPTION
**Pull request by marc-hb**
See commit messages

### CODE REVIEW

**Comment by marc-hb** [comment]
A relatively high number (3) of errors in https://sof-ci.01.org/softestpr/PR1073/build534/devicetest/index.html but they're all unrelated.

https://sof-ci.01.org/softestpr/PR1073/build533/devicetest/index.html and https://sof-ci.01.org/softestpr/PR1073/build535/devicetest/index.html have the usual amount of suspend/resume failures.

File: case-lib/lib.sh [hunk 1]
```diff
@@ -711,7 +709,10 @@ is_zephyr()
 get_firmware_path()
 {
     journalctl_cmd -k |
-        awk '/sof.*request_firmware/ { sub(/^.*request_firmware/,""); last_loaded_file=$1 } END { print last_loaded_file }'
+        awk '/sof.*[[:blank:]]request_firmware[[:blank:]]/ {
+               sub(/^.*request_firmware/,""); last_loaded_file=$1
+             }
+         END { print last_loaded_file }'
```
Comment (greg-intel) [inline case-lib/lib.sh:715]: Was this broken before?

**Review by greg-intel — APPROVED** [review]

File: case-lib/lib.sh (hunk 1 above)
Comment (marc-hb) [reply case-lib/lib.sh:715]: It was broken on my system that has some extra logs, check the commit message.

Grepping logs is "broken by design". In this case the real fix will be https://github.com/thesofproject/linux/issues/3867 but that's not at the top of my TODO list right now.

A more restrictive regex is always "LESS broken by design" - until someone changes the format of the log message. Unlikely here but possible.

**Comment by marc-hb** [comment]
Thanks @greg-intel for the review.

This one is really small, very low-risk, well tested and easy to revert: doing the wrong thing and merging it on a Friday.

**Pull request merged**